jdk-annotations/jdk-integration-test/src/test/java/com/uber/nullaway/jdkannotations/JDKIntegrationTest.java

```diff
@@ -329,4 +329,26 @@ public void genericParameterTest() {
             "}")
         .doTest();
   }
+
+  @Test
+  public void libraryLoadMethodParser() {
+    compilationHelper
+        .setArgs(
+            Arrays.asList(
+                "-d",
+                temporaryFolder.getRoot().getAbsolutePath(),
+                "-XepOpt:NullAway:AnnotatedPackages=com.uber",
+                "-XepOpt:NullAway:JarInferEnabled=true"))
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import org.jspecify.annotations.Nullable;",
+            "import java.util.List;",
+            "class Test<T> {",
+            "  @Nullable List<? super T> getMap() {",
+            "    return null;",
+            "  }",
```

Comment:
This is not the right place to put the test.  You should add this method to `ReturnAnnotation` in `test-annotated` and `test-unannotated` and then call the method here, and make sure the annotation on the return type gets through.  I would expect that to not work without your fix.